app/src/layouts/kanban/kanban.vue

```diff
@@ -0,0 +1,397 @@
+<template>
+	<div class="kanban">
+		<draggable
+			:model-value="groupedItems"
+			group="groups"
+			item-key="id"
+			draggable=".draggable"
+			:animation="150"
+			:disabled="groupSortField === null"
+			class="draggable"
+			@change="changeGroupSort"
+		>
+			<template #item="{ element: group }">
+				<div class="group" :class="{ draggable: group.id !== null }">
+					<div class="header">
+						<div class="title">
+							<div class="title-content">
+								{{ group.id === null ? t('layouts.kanban.no_group') : group.title }}
+							</div>
+							<span class="badge">{{ group.items.length }}</span>
+						</div>
+						<div v-if="group.id !== null" class="actions">
+							<router-link :to="`${collection}/+`"><v-icon name="add" /></router-link>
+							<v-menu show-arrow placement="bottom-end">
+								<template #activator="{ toggle }">
+									<v-icon name="more_horiz" clickable @click="toggle" />
+								</template>
+
+								<v-list>
+									<v-list-item clickable @click="openEditGroup(group)">
+										<v-list-item-icon><v-icon name="edit" /></v-list-item-icon>
+										<v-list-item-content>{{ t('layouts.kanban.edit_group') }}</v-list-item-content>
+									</v-list-item>
+									<v-list-item class="danger" clickable @click="deleteGroup?.(group.id)">
```

Comment:
Im wondering if we should add a default function stub to this prop instead of the optional chaining function call 🤔

Reply:
In hindsight, I've opted to remove the optional chaining because I've marked this prop function to no longer be optional in the initial commit (it was optional in the original repo where it resided in), especially since functions in props for other layouts aren't marked optional either. 👍